files/image_config/platform/rc.local

```diff
@@ -328,10 +328,10 @@ if [ -f $FIRST_BOOT_FILE ]; then
         mv /host/grub.cfg /host/grub/grub.cfg
     fi
 
+    # Create dir where following scripts put their output files
+    mkdir -p /var/platform
```

Comment:
why do we create it?

Reply:
We have created this to support system related queries and to support platform specific validations. We will remove this dependency in the future once the system calls are serviced through host dbus implementation.